redis/src/test/scala/zio/redis/ApiSpec.scala

```diff
@@ -268,6 +271,354 @@ object ApiSpec extends BaseSpec {
             } yield assert(hash)(equalTo(RedisType.Hash))
           }
         )
+      ),
+      suite("lists")(
+        testM("lIndex first element") {
+          for {
+            key   <- uuid
+            _     <- lPush(key)("world", "hello")
+            index <- lIndex(key, 0l)
```

Comment:
Please use the uppercase L everywhere, using the lowecase makes it very difficult to read.